src/strings/ops.c

```diff
@@ -509,6 +518,13 @@ MVMString * MVM_string_repeat(MVMThreadContext *tc, MVMString *a, MVMint64 count
     if (agraphs == 0)
         return tc->instance->str_consts.empty;
 
+    /* Total size of the resulting string can't be bigger than an MVMString is allowed to be. */
+    total_graphs = (MVMuint64)agraphs * (MVMuint64)count;
```

Comment:
agraphs can't be greater than 0xFFFFFFFF (since num_graphs is a MVMuint32) and we already check that count is not greater than (1 << 30). Should be fine, right?